tenant-cd/src/main/java/ai/vespa/hosted/cd/ProductionTest.java

```diff
@@ -1,5 +1,5 @@
 // Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
-package com.yahoo.vespa.tenant.cd;
+package ai.vespa.hosted.cd;
```

Comment:
I'm surprised! Is this something we should generally be doing in new code?

Reply:
That's what I was told :)